src/main/resources/application.properties

```diff
@@ -1 +1,9 @@
+helpdeskService.uri=${HELPDESK_SERVICE_URI}
+helpdeskService.readTimeout=${HELPDESK_SERVICE_READ_TIMEOUT}
```

Comment:
is this file used? maybe it's a typo from the first configuration with springboot?